content/docs/command-reference/remote/add.md

```diff
@@ -18,14 +18,21 @@ positional arguments:
 
 ## Description
 
-`name` and `url` are required. `url` specifies a location to store your data. It
-can represent a cloud storage service, an SSH server, network-attached storage,
-or even a directory in the local file system. (See all the supported remote
-storage types in the examples below.) If `url` is a relative path, it will be
-resolved against the current working directory, but saved **relative to the
-config file location** (see LOCAL example below). Whenever possible, DVC will
-create a remote directory if it doesn't exist yet. (It won't create an S3 bucket
-though, and will rely on default access settings.)
+`name` and `url` are required. The `name` is used to identify the remote and
+must be unique for the project.
+
+`url` specifies a location to store your data. It can represent a cloud storage
```

Comment:
Unrelated (most of the file changes)

Reply:
Well, they were triggered by the "must" instances here. Could be left in the PR, it's a good improvement, I think.